src/kixi/heimdall/handler.clj

```diff
@@ -1,20 +1,42 @@
 (ns kixi.heimdall.handler
   (:require [compojure.core :refer :all]
             [compojure.route :as route]
-            [ring.middleware.defaults :refer [wrap-defaults site-defaults]]
+            [ring.middleware.defaults :refer [wrap-defaults api-defaults]]
             [buddy.hashers :as hs]
-            [kixi.heimdall.user :as user]
             [kixi.heimdall.application :as app]
             [qbits.alia :as alia]
-            ))
+            [taoensso.timbre :as log]))
+
+(defn find-user
+  [password-hash username]
+  {:username username :password-hash password-hash})
+
+(defn add-user!
+  []
+  true)
+
+(defn auth-user [ds credentials]
```

Comment:
Move the domain functions to a separate ns and just have the compojure/ring handling here?